### PR TITLE
Add kafka downstream example

### PR DESCRIPTION
**Pull request by chenmin1992**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add kafka downstream example for convenience, and change the dirs of pumps to a better path that correspond to its default port.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [x] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@chenmin1992 Could you please sign the [CLA](https://github.com/pingcap/docs-cn/pull/6416#issuecomment-856476264) for this PR to be merged? Thanks

**Review by july2993 — APPROVED** [review]

File: config-templates/complex-tidb-binlog.yaml [hunk 1]
```diff
@@ -92,42 +92,58 @@ pump_servers:
...
       syncer.db-type: "tidb"
       syncer.to.host: "10.0.1.12"
       syncer.to.user: "root"
       syncer.to.password: ""
       syncer.to.port: 4000
+  - host: 10.0.1.13
+    port: 8249
+    deploy_dir: "/tidb-deploy/drainer-8249"
+    data_dir: "/tidb-data/drainer-8249"
+    # If drainer doesn't have a checkpoint, use initial commitTS as the initial checkpoint.
```
Comment (TomShawn) [inline config-templates/complex-tidb-binlog.yaml:135]:
```suggestion
    # If Drainer does not have a checkpoint, use the initial commitTS as the initial checkpoint.
```

File: config-templates/complex-tidb-binlog.yaml [hunk 2]
```diff
@@ -92,42 +92,58 @@ pump_servers:
...
+    # Will get a latest timestamp from pd if commit_ts is set to -1 (the default value).
```
Comment (TomShawn) [inline config-templates/complex-tidb-binlog.yaml:136]:
```suggestion
    # If commit_ts is set to -1 (the default value), you will get a latest timestamp from PD.
```

File: config-templates/complex-tidb-binlog.yaml [hunk 3]
```diff
@@ -92,42 +92,58 @@ pump_servers:
...
+    commit_ts: -1
+    # The following configs are used to overwrite the `server_configs.drainer` values.
```
Comment (TomShawn) [inline config-templates/complex-tidb-binlog.yaml:138]:
```suggestion
    # The following configurations are used to overwrite the `server_configs.drainer` values.
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**